ruby-rails-blog/PKGBUILD

```diff
@@ -17,8 +17,8 @@ depends=(
 
 build() {
     cd ${startdir}
-    RAILS_ENV=production bin/bundle install --deployment --without development test
-    RAILS_ENV=production bin/rails assets:precompile db:migrate
+    bundle install --path gems --deployment --without development test
```

Comment:
the `path` arg makes it so that all that gems are installed into a local (version-independent) directory (named "gems" in this case). This should reduce ruby-version-related headaches.

As for `bundle install`, UBOS already puts bundle in /usr/bin/bundle, which I would think would be generally preferable to having developers maintain their own copy.

Reply:
(and the rails environment won't matter to bundle install)

Reply:
Sounds excellent to me. Just one thing: please increment the `pkgrel` as part of the PR so it will actually get rebuilt.